easybuild/easyconfigs/o/OpenSlide/OpenSlide-3.4.1-GCCcore-8.2.0-largefiles.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenSlide'
+version = '3.4.1'
+versionsuffix = '-largefiles'
+
+homepage = 'https://openslide.org/'
+description = """OpenSlide is a C library that provides a simple interface to
+read whole-slide images (also known as virtual slides)."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+github_account = 'openslide'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_large_file_support.patch']
+checksums = [
+    'a5d869916e370125421535dcce778b2ba625dc50d920aa4ca93bbaaa6a7b470c',  # v3.4.1.tar.gz
+    'cb618053f4ae6c3ce37d1b8b0e4ef7c55fd17378776d13be4aa4efab91706b8c',  # OpenSlide-3.4.1_large_file_support.patch
+]
+
+# See https://github.com/openslide/openslide/pull/276 - adds large file support
```

Comment:
misplaced/leftover comment, can be dropped now that it's included in the patch itself i think.